scenarios/barnoux/coraza.yaml

```diff
@@ -0,0 +1,15 @@
+type: trigger
+name: barnoux/coraza
+description: "Web exploitation detected via coraza"
+filter: evt.Meta.log_type == 'modsecurity' && evt.Parsed.ruleid == '949110'
```

Comment:
rather being coraza maybe should be named something to do with CRS?

Reply:
yes you right, i will try to find a better name.